### ISSUE TITLE
Problems with "uk" naming in translations

### ISSUE THREAD

**Issue by e-yakubov** [laravel/nova-issues#3323]
- Laravel Version: 8.33.1
- Nova Version: 3.22.0
- PHP Version: 7.4.16
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#

### Description:
Trying to create Ukrainian translations on Laravel Nova. 
I have created the file : "resources/lang/vendor/nova/uk.json", and added translations for defaults ,actions, my own resources, etc. But the system doesn't use the translations correctly. For example "Attach :resource", the system translates only "Attach", but resource remains the same, at the same time, "Create :resource" is not being translated at all. 
But if I use the file with name "ua.json",with same context it works properly. 

used file uk.json
<img width="1179" alt="Screenshot 2021-04-03 at 23 55 42" src="https://user-images.githubusercontent.com/5730844/113491262-22a4bb00-94d8-11eb-928e-ad00661e020b.png">
<img width="1169" alt="Screenshot 2021-04-03 at 23 55 25" src="https://user-images.githubusercontent.com/5730844/113491263-23d5e800-94d8-11eb-84aa-7baf1cfc1612.png">

used file ua.json
<img width="1186" alt="Screenshot 2021-04-03 at 23 54 46" src="https://user-images.githubusercontent.com/5730844/113491264-23d5e800-94d8-11eb-8f00-d744187dbdba.png">

**Comment by crynobone** [comment]
This definitely hard to follow, can you recreate a full reproducing repo and post it on github (without Nova source) so I can review it. See https://github.com/nova-issues for references.
